### PR TITLE
Hotdog

### PR DESCRIPTION
**Pull request by ghost**
:cl: More Robust Than You
add: HOTDOGS. CRAFT A BUN AND SAUSAGE FOR MYSTERY HOT DOG GOODNESS
/:cl:

hotdog2

### CODE REVIEW

File: code/modules/food_and_drinks/food/snacks_meat.dm [hunk 1]
```diff
@@ -103,6 +103,15 @@
 	bonus_reagents = list("nutriment" = 1, "vitamin" = 1)
 	list_reagents = list("nutriment" = 6, "vitamin" = 1)
 	tastes = list("meat" = 1)
+	
+/obj/item/weapon/reagent_containers/food/snacks/hotdog
+	name = "hotdog"
+	desc = "HOTDOG!!"
```
Comment (Niknakflak) [inline code/modules/food_and_drinks/food/snacks_meat.dm]: Something more descriptive than this surely

**Comment by ninjanomnom** [comment]
That commit history gave me a new disease doctors have yet to name

**Comment by MoreRobustThanYou** [comment]
earlier than 18b0922 is just me updating master. no idea why it shows up

File: code/modules/food_and_drinks/food/snacks_meat.dm (hunk 1 above)
Comment (MoreRobustThanYou) [inline code/modules/food_and_drinks/food/snacks_meat.dm]: i'm trying to sneak in a silicon valley reference here

**Comment by fludd12** [comment]
Squaaaash

**Comment by Fox-McCloud** [comment]
Should update your master *first* before commiting, not the other way around.

**Comment by ExcessiveUseOfCobblestone** [comment]
hotdog

**Review by Cyberboss — DISMISSED** [review]

**Comment by Deputi** [comment]
that's a nice looking wiener you got there, but there's already a hot dog recipe. maybe you can replace that one with yours
hot diggity dog

**Comment by Cyberboss** [comment]
What is this PR even changing now?

**Comment by MoreRobustThanYou** [comment]
i dunno

**Comment by WJohn** [comment]
wait don't we have these already??

**Comment by MoreRobustThanYou** [comment]
i dunno. i changed the sprite

**Comment by DoctoreMad** [comment]
Yes we already have hotdogs
What the fuck is the PR maker doing?

**Comment by MoreRobustThanYou** [comment]
not the slightest clue

**Pull request closed without merging**